### DISCUSSION TITLE
Microsoft Store apps goes missing with W10 Start Menu.

### DISCUSSION THREAD

**Discussion by dAycicek** [valinet/ExplorerPatcher#1579] [Q&A]
Whenever I choose W10 as Start Menu style, all Microsoft Store-related apps such as Calculator or Minecraft Launcher goes missing from Start Menu > All Apps. I hate that because I usually use start menu to access apps because I pin them. Can someone please help?
(When I write their name of course they appear but still I can't pin them and I can't see them in all apps section.)

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1108

**Comment by vippium** [comment]
> Whenever I choose W10 as Start Menu style, all Microsoft Store-related apps such as Calculator or Minecraft Launcher goes missing from Start Menu > All Apps. I hate that because I usually use start menu to access apps because I pin them. Can someone please help? (When I write their name of course they appear but still I can't pin them and I can't see them in all apps section.)



Here is the solution. But it has a bug that "Pin to Start" won't work. Alternate is also provided there by me.

https://github.com/valinet/ExplorerPatcher/issues/1534#issuecomment-1601017536

Proof: 
![2023-06-22_20h05_58](https://github.com/valinet/ExplorerPatcher/assets/93650504/a687a9e1-7f54-43bf-95d1-abf640336b04)
